JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -115,16 +115,16 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
+                                    "firstSecond" = gettextf("Benford's law states that in many naturally occurring collections of numerical observations, the leading significant digit is likely to be small. The goal of this procedure is to determine to what extent the first two leading digits in the data set follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that are not distributed according to Benford's law might need further investigation.", confidenceLabel),
+                                    "last" = gettextf("Benford's law states that in many naturally occurring collections of numerical observations, the leading significant digit is likely to be small. The goal of this procedure is to determine to what extent the last digits in the data set follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that are not distributed according to Benford's law might need further investigation.", confidenceLabel))
     
     } else if (options[["distribution"]] == "uniform"){
 
       procedureText <- base::switch(options[["digits"]],
-                                    "first" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to which extent the first leading numbers in the population follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do not conform to the uniform distribution might need further verification.", confidenceLabel),
-                                    "firstSecond" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to which extent the first two leading numbers in the population follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do not conform to the uniform distribution might need further verification.", confidenceLabel),
-                                    "last" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to which extent the last numbers in the population follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do not conform to the uniform distribution might need further verification.", confidenceLabel))
+                                    "first" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to what extent the first leading digits in the data set follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do are not uniformly distributed might need further investigation.", confidenceLabel),
```

Comment:
same confidence

Reply:
"The uniform distribution assigns equal probability to all possible first digits that may occur. The goal of this procedure is to determine to what extent the first leading digits in the data set follow the uniform distribution, and to test this relation with a type I error of <b>%1$s</b>. If the uniform distribution assumption is reasonable/desirable, then violation of uniformity is cause for further investigation."